### DISCUSSION TITLE
Has anyone tried using this on ubuntu image with node 15?

### DISCUSSION THREAD

**Discussion by arupadhy** [uNetworking/uWebSockets.js#487] [General]
I am evaluating some socket libraries for our use cases with node. Has anyone tried using this lib with node 15 on ubuntu_focal?

**Comment by joshxyzhimself** [comment]
Yep got ubuntu desktop focal fossa with node 14, ubuntu servers focal fossa node 15, this library works great. Version v19.x.x was recently released too, I think that one includes binaries for node 16 along with some other changes

https://github.com/uNetworking/uWebSockets/releases/tag/v19.0.0
